examples/js/exporters/GLTFExporter.js

```diff
@@ -448,11 +448,7 @@ THREE.GLTFExporter.prototype = {
 		 */
 		function processImage( map ) {
 
-			if ( cachedData.images[ map.uuid ] !== undefined ) {
-
-				return cachedData.images[ map.uuid ];
-
-			}
+			// @QUESTION Needs image cache for the case where two different textures share the same image?
```

Comment:
Could we keep this cache, maybe using a `Map`? They're supported in IE11+

```js
if ( cachedData.images.has( map.image ) ) {

  return cachedData.images.get( map.image );

}
```

Reply:
I'm thinking to switch from `Object` to `WeakMap` for cache like `WebGLProperties` does soon. How about recovering image cache when we'll do?

Reply:
Ok sure. Would just call this `@TODO` instead of `@QUESTION` then. 👍 But I think Map/WeakMap should be no different for this use case.

Reply:
Well, it's still question for me. I'm really not sure if two or more textures share the same image tho Three.js allows so.

Reply:
That would happen with [texture atlas transforms](https://github.com/mrdoob/three.js/compare/dev...donmccurdy:feat-gltfloader-texturetransform), for example.

Reply:
OK, I've replaced because we'll have actual use case.